packages/destination-actions/src/destinations/fullstory/identifyUserV2/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { createUserRequestParams } from '../request-params'
+import { normalizePropertyNames } from '../vars'
+
+const action: ActionDefinition<Settings> = {
+  title: 'Identify User V2',
+  description: 'Sets user identity variables, creates user if it does not exist',
```

Comment:
Nit: This reads a little odd to me ("user"/"it"). Could go with something like:

"Sets user identity variables. Creates a FullStory user if a user matching the given uid doesn't already exist."